debian/rules

```diff
@@ -39,6 +39,8 @@ override_dh_auto_build:
 override_dh_auto_test:
 	export LD_LIBRARY_PATH=${ROOT_DIR}/build/gst/nnstreamer:${ROOT_DIR}/build/gst/nnstreamer/tensor_filter
 	export GST_PLUGIN_PATH=${ROOT_DIR}/build
+	export NNSTREAMER_FILTERS=${ROOT_DIR}/build/ext/nnstreamer/tensor_filter
```

Comment:
.spec and debian rules,
LD_LIBRARY_PATH also need update with NNSTREAMER_FILTERS and NNSTREAMER_DECODERS

Reply:
No you don't need it. ```ldopen``` does not require to have LD_LIBRARY_PATH.

Reply:
I see, then remove :${ROOT_DIR}/build/gst/nnstreamer/tensor_filter